include/dxc/DXIL/DxilResource.h

```diff
@@ -29,8 +29,19 @@ class DxilResource : public DxilResourceBase {
   static unsigned GetNumDimensionsForCalcLOD(Kind ResourceKind);
   /// Total number of offsets (in [-8,7]) necessary to access resource.
   static unsigned GetNumOffsets(Kind ResourceKind);
-  /// Whether the resource kind is texture.
+  /// Whether the resource kind is a texture. This does not include
+  /// FeedbackTexture.
   static bool IsAnyTexture(Kind ResourceKind);
+  /// Whether the resource kind is an array of textures. This does not include
+  /// texture feedback arrays.
+  static bool IsAnyTextureArray(Kind ResourceKind);
```

Comment:
Can we either remove `IsAnyTextureArray`, or rename it to `IsAnyTextureWithArrayDimension` or `IsArrayTexture`?

the `IsArrayTexture` option doesn't feel ambiguous to me because it's the way such texture types are referred to like: "this is an array texture", which is different than "array of textures".